addon/templates/components/single-event.hbs

```diff
@@ -93,6 +93,12 @@
   <fieldset>
     <div class="single-event-learningmaterial-list">
       <h2>{{t "general.sessionLearningMaterials"}}</h2>
+      {{#if (and this.currentUser.userIsStudent (await this.courseId))}}
```

Comment:
`userIsStudent` returns a promise. Needs to be wrapped in an `await`.

Reply:
👍